src/catalogue/app/src/main/java/catalogue/db/DbSetup.java

```diff
@@ -44,7 +44,7 @@ public DbSetup(DataSource dataSource,
 
     @EventListener
     @Transactional
-    void init(StartupEvent startupEvent) {
+    void onStartupEvent(StartupEvent startupEvent) {
```

Comment:
FYI, event listeners methods can have any name. This is just a convention (so I agree with the change), but what really matters is `@EventListener`

Reply:
okay I see